docs/en/user/installation.rst

```diff
@@ -151,22 +188,58 @@ Assuming you install wallabag in the /var/www/wallabag folder, here's the recipe
 
 After reloading or restarting nginx, you should now be able to access wallabag at http://domain.tld.
 
-.. note::
+.. tip::
 
     When you want to import large file into wallabag, you need to add this line in your nginx configuration ``client_max_body_size XM; # allows file uploads up to X megabytes``.
 
+Configuration on LigHTTPd
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Edit your ``lighttpd.conf`` file and paste this configuration into it:
+
+::
+
+    server.modules = (
+    "mod_fastcgi",
+    "mod_access",
+    "mod_alias",
+    "mod_compress",
+    "mod_redirect",
+    "mod_rewrite",
```

Comment:
You should add one level of indentation here, same for `url.rewrite-if-not-file` below